gabbi/httpclient.py

```diff
@@ -182,5 +182,5 @@ def get_http(verbose=False, caption=''):
         if verbose == 'headers':
             body = False
         return VerboseHttp(body=body, headers=headers, colorize=colorize,
-                           stream=stream, caption=caption)
-    return Http()
+                           stream=stream, caption=caption, strict=True)
+    return Http(strict=True)
```

Comment:
FWIW, while I'm usually a proponent of return-early abortion, in this case it actually confused me because
a) the abort condition, and thus relevant context, didn't fit into the diff (or mental buffer, for that matter), so I was briefly puzzled as to why there were two return statements at different levels of indentation
b) the abort block is significantly _more_ complex than the regular path

Thus I'd either flip the condition (with some reluctance because negotiation is a little awkward) despite some hesitation or use `else` (though (b) would still apply there).

Reply:
Feel free to make that adjustment in a later patch. I don't want to do so in this change.

Reply:
Makes sense - which do we prefer:
```python
if not verbose:
    return return Http(strict=True)

headers = False if verbose == 'body' else True
body = False if verbose == 'headers' else True
return VerboseHttp(…)
```
or
```python
if verbose:
    headers = False if verbose == 'body' else True
    body = False if verbose == 'headers' else True
    return VerboseHttp(…)
else:
    return return Http(strict=True)
```

Reply:
No strong preference, but probably the former a bit more, because of less indentation. Thanks.